src/components/dev-hub/feedback/feedback-container.js

```diff
@@ -11,21 +18,111 @@ const feedbackItems = graphql`
     }
 `;
 
-const FeedbackContainer = () => {
-    const data = useStaticQuery(feedbackItems);
+const StyledButtonContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+    margin-bottom: 4px;
+`;
+
+const StyledForm = styled('form')`
+    padding: 0 ${size.medium};
+
+    button,
+    label {
+        margin: ${size.default} 0;
+    }
+
+    ${InputContainer} {
+        margin: ${size.default} 0;
+    }
+`;
+
+const StyledDescription = styled(P)`
+    display: inline-block;
+    font-family: Akzidenz;
+    font-size: ${fontSize.small};
+    line-height: ${lineHeight.small};
+`;
+
+const StyledHeader = styled(H5)`
+    margin-bottom: 0;
```

Comment:
There is a `collapse` prop for `H5` which also does this so we can use that here